Src/WitsmlExplorer.Api/Workers/Delete/DeleteEmptyMnemonicsWorker.cs

```diff
@@ -165,11 +165,17 @@ private ICollection<LogCurveInfo> FindNullMnemonics(double nullDepthValue, DateT
             {
                 if (logToCheck.IndexType == WitsmlLog.WITSML_INDEX_TYPE_MD)
                 {
-                    nullMnemonics.AddRange(logCurves.Where(logCurve => logCurve.MinDepthIndex == nullDepthValueString && logCurve.MaxDepthIndex == nullDepthValueString));
+                    nullMnemonics.AddRange(logCurves.Where(logCurve =>
+                        (logCurve.MinDepthIndex == nullDepthValueString || logCurve.MinDepthIndex == null) &&
+                        (logCurve.MaxDepthIndex == nullDepthValueString || logCurve.MaxDepthIndex == null)
```

Comment:
I think we should include deleting null values as an option in the "Delete empty mnemonics" modal. Currently, we are deleting mnemonics with no max and min index regardless of which "Null time value" or "Null depth value" we are setting.
![image](https://github.com/equinor/witsml-explorer/assets/48485965/a7c799aa-d5d8-4b96-abd0-ca929d1b0ac2)

Reply:
Fixed.